src/containers/EditSubjectFrontpage/components/SubjectpageArticles.tsx

```diff
@@ -45,7 +45,8 @@ const SubjectpageArticles: FC<Props & tType> = ({ t, editorsChoices, elementId,
       } else {
         newArticle = await fetchDraft(article.id);
       }
-      const temp = [...articles, newArticle];
+      // @ts-ignore  Temporary ugly hack for mismatching Article types, should be fixed when ConceptForm.jsx -> tsx
```

Comment:
er litt fedd for at disse ikke blir gjort. Burde hvertfall slenge på en TODO, men hadde du noen planer om å gjøre dette i nærmeste fremtid? :-) / lage en issue på det?

Reply:
Altså vi nærmer oss mer og mer det å ha typescript på plass i alle filer. Vi kommer til å nå et punkt hvor dersom vi har definert f.eks ArticleType på 2 forskjellige måter så vil typescript klage der det kræsjer, men siden vi har fortsatt blanda js med ts så vil ikke alle stedene ha typesjekk på dette. Min tanke er å utsette type-fiksen til vi faktisk når en større dekning av typescript rett og slett fordi da vil typescript klage når vi gjør en feil, i motsetning til js som vi har nå.

Det kreves også større planlegging på hvordan den endelige typen kommer til å se ut og få alle komponenter til å være på samme lag. Krever større rewrite av TopicArticlesForm/LearningResourceForm (formik) og ConceptForm til å få alt riktig på plass. Og det er ikke helt det optimale å gjøre akkurat nå i denne pull-requesten :sweat_smile: 

Enig med TODO tho, blir kanskje mer synlig og folk ikke glemmer det helt bort :crossed_fingers: